sdks/python/src/opik/integrations/aisuite/opik_tracker.py

```diff
@@ -23,6 +24,10 @@ def track_aisuite(
     Returns:
         The modified AISuite client with Opik tracking enabled.
     """
+    # Check if tracing is active
+    if not is_tracing_active():
```

Comment:
No integration should have this logic when `trace_*` is called. The enable/disable tracing logic should live only in base_track_decorator.